app/views/admins/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="container">
+  <h1>Admin Dashboard</h1>
+  <p>First name: <%= @admin.first_name %></p>
+  <p>Last name: <%= @admin.last_name %></p>
+  <p>Email: <%= @admin.email %></p>
+  <p><%= link_to "Edit Profile", edit_admin_path(@admin) %></p>
+</div>
+
+<h2>Order History for all Customers</h2>
+
+<h3>Order summary</h3>
+<ul>
+<% @statuses.sort_by_name.each do |status| %>
+  <li class="status_<%= status.name.downcase %>">
```

Comment:
Do you need this downcase?   I think the statuses are not capitalized when they're saved.